src/spyglass/histology/v1/histology.py

```diff
@@ -0,0 +1,141 @@
+    definition = """
+    # Represents a single histology preparation for a subject
+    -> Subject
+    histology_id: varchar(32)           # User-defined ID (e.g., 'probe_track_run1')
+    ---
+    prep_date=NULL: date                # Optional: Date of tissue preparation
+    slice_orientation: enum(            # Orientation of sections
+        "coronal",
+        "sagittal",
+        "horizontal",
+        "other"
+    )
+    slice_thickness: float              # (um) Thickness of sections
+    mounting_medium=NULL: varchar(128)  # e.g., 'DPX', 'Fluoromount-G'
+    experiment_purpose: varchar(1024)   # e.g., 'Probe track recovery', 'Anatomical ref'
+    notes="": varchar(2048)             # Optional general notes about the preparation
+    -> [nullable] LabMember.proj(histology_experimenter='user_name') # Optional: who did the prep?
+
+    # --- Data Source ---
+    output_format='TIFF stack': varchar(64) # Format of raw image data from scanner
+    raw_scan_path: varchar(512)         # Path to the raw output (e.g., folder containing TIFF stack)
+    """
+
+    class HistologyStain(SpyglassMixin, dj.Part):
+        definition = """
+        # Details of specific stains used in a histology preparation
+        -> Histology
+        stain_index: tinyint unsigned    # Use index for multiple stains (0, 1, 2...)
+        ---
+        identified_feature: varchar(128) # Biological target/marker (e.g., 'Nissl Bodies', 'ChR2-tdTomato+', 'ProbeTrack_DiI')
+        visualization_agent: varchar(128)# Method/molecule making feature visible (e.g., 'Cresyl Violet', 'Native tdTomato', 'DiI', 'Alexa 488')
+        stain_type: enum(                # Type of staining method used
+            "immunohistochemistry",
+            "genetic_marker",
+            "tracer",
+            "anatomical",
+            "histochemical",
+            "in_situ_hybridization",
+            "other"
+        ) = "anatomical"
+        stain_protocol_name=NULL: varchar(128) # Optional: name of the protocol used
+        antibody_details=NULL: varchar(255)    # Optional: specific antibody info (e.g. company, cat#, lot#)
+        stain_notes="": varchar(1024)          # Optional notes about this specific stain (e.g., concentration)
+        """
+
+
+@schema
+class HistologyImages(SpyglassMixin, dj.Computed):
+    definition = """
+    # Links Histology info to the Analysis NWB file containing the image data
+    -> Histology
+    images_id: varchar(32)              # User-defined ID for these images (e.g., histology_id)
+    ---
+    -> AnalysisNwbfile                  # Link to the NWB file storing image data
+    processing_time=CURRENT_TIMESTAMP: timestamp # Timestamp of NWB file creation
+    # --- Image Acquisition/Processing Details ---
+    color_to_stain=NULL: blob           # Mapping channel colors to stain features (e.g., {'DAPI': 'blue', 'GFAP': 'green'})
+    pixel_size_x: float                 # (um) Pixel size in X dimension after processing/scaling
```

Comment:
Is it possible that the post-processing pixel size will differ across images in a stack?